### PR TITLE
Removed cron for aro and reverting some of the previous changes

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Changes:
* Remove cron for ARO job to leave it as a POC.
* Revert the iterations multiplier to default `9`.
* Revert the binary to nightly for GCP.

### CODE REVIEW

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-net-perf

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
 - dockerfile_path: prow/Dockerfile
   to: ocp-qe-perfscale-ci-gcp-413
-promotion:
-  namespace: ci
-  tag: latest
 releases:
   latest:
-    release:
-      channel: stable
+    candidate:
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml:11]: when was this changed introduced? Can we change this in a different pr

**Comment by svetsa-rh** [comment]
@vishnuchalla I do want us to remove this chain https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml#L66C52-L66C52 and explicitly split each workload tests as we currently do in rosa. Until we make sure that each workload results are satisfactory over a period of time, I don't think we should run them in a chain like you did here. Especially for ARO managed services, this is all new and we haven't analyzed results. We can always bundle them up at a later time depending on our experiences.

**Comment by svetsa-rh** [comment]
@vishnuchalla Also, after you split the tests, please keep cron for c-d-v2, n-d and n-d-h. We can run each test on a bi-weekly basis and rotate the cadence among tests.

**Pull request closed without merging**